pkg/envoy/agent.go

```diff
@@ -149,6 +149,12 @@ func (a *Agent) terminate() {
 		ticker := time.NewTicker(activeConnectionCheckDelay)
 		for range ticker.C {
 			ac := a.activeProxyConnections()
+			if ac == -1 {
+				log.Info("downstream_cx_active are not available. This either means there are no downstream connection established yet" +
+					" or the stats are not enabled. Skipping active connections check...")
```

Comment:
In theory it could be an error too

Reply:
I did not change it to error because no client might have connected to it.

Reply:
I mean it could be an error calling 127.0.0.1:15000

Reply:
I see. Handled error differently